txnkv/transaction/2pc.go

```diff
@@ -1549,6 +1549,7 @@ func (c *twoPhaseCommitter) execute(ctx context.Context) (err error) {
 		if err = c.txn.GetMemBuffer().FlushWait(); err != nil {
 			return err
 		}
+		c.txn.pipelinedCancel()
```

Comment:
Is there a difference to cancel here?

Reply:
No difference, just avoid leaving uncanceled context.